ansible/roles/controller/tasks/deploy.yml

```diff
@@ -272,6 +272,11 @@
 
       "CONFIG_whisk_controller_activation_pollingFromDb": "{{ controller_activation_pollingFromDb | default(true) | lower }}"
 
+      "CONFIG_whisk_etcd_hosts": "{{ etcd_connect_string }}"
```

Comment:
Since ShardingContainerPoolBalancer does not use etcd, should we selectively configure these configurations?

Reply:
In the case of configuration, it would be good to add only what is needed when etcd is used in the controller & invoker. I will exclude it from this PR.